php/libraries/Settings.class.inc

```diff
@@ -61,7 +61,7 @@ class Settings
             return $url;
         }
         $www = $this->_config->getSetting('www');
-        return $www['url'];
+        return isset($www['url']) ? $www['url'] : '';
```

Comment:
whats this change for ?

Reply:
Loris won't work for my local setup without it. It just makes null become an empty string and so no error is thrown. @ridz1208

Reply:
it's an unecessary confusing change, you should find a way to fix it locally, or just remove it before submiting the PR please

Reply:
@ridz1208 tried having something in the database where that url gets retrieved and it makes everything break. Unsure why but I can remove when everything is ready to be merged.